index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <header class="app-bar">
+        <nav class="lower-bar">
+          <p class="logoo">Somarib Emmanuel</p> 
+          <div class="menu-icons"> <img class="menuimg" src="image/menu.png" alt="menu icon"></div>
+        </nav>
+      </header>
+
+      <section class="bgimage">
+        <h1 class="myname">Somarib Emmanuel</h1>
+        <h2 class="jobtitle">Software Developer</h2> 
+          <p class="abilities">
+            I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,  
+            don’t hestiate to contact me.
+          </p>
+        <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+        <p class="frame38"><img src="image/Frame38.png" alt="language proficiency"></p>
+        <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+        <p class="social"><img src="image/Social_media.png" alt="social media links"></p>
```

Comment:
- [ ] The technologies lists and the social links should be written by you using `<ul>` tags and `<li>` tags for the items. The social links should be links for your real pages so `<a>` tags should be used to make the user connect to your Github, Twitter, etc. Kindly unwrap these images as the previous Code Reviewer asked.